includes/payment_methods.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+class Migrator_CLI_Payment_Methods {
+
+	public function import_stripe_data_into_woopayments() {
+
+		Migrator_CLI_Utils::health_check();
+
+		if ( ! class_exists( 'WC_Payments' ) ) {
+			WP_CLI::line( WP_CLI::colorize( '%RWooPayments is not active' ) );
+			die();
+		}
+
+		$request = \WCPay\Core\Server\Request::get( WC_Payments_API_Client::CUSTOMERS_API );
+		$result = $request->send();
+		$stripe_customers = $result['data'];
+
+		$this->import_customers_data( $stripe_customers );
+		WP_CLI::line( 'Done' );
+	}
+
+	private function import_customers_data( $stripe_customers ) {
+		foreach ( $stripe_customers as $stripe_customer ) {
+
+			$user = get_user_by('email', $stripe_customer['email'] );
```

Comment:
```suggestion
			$user = get_user_by( 'email', $stripe_customer['email'] );
```

Minor - missing space after bracket.